### DISCUSSION TITLE
Problem with homematic homebridge addin

### DISCUSSION THREAD

**Discussion by ppoths2297** [npm/feedback#144] [General]
Hi,

I use the homematic addin in my homebridge via the homematic ccu3. I added the homematic weatherstation pro and in the homematic ccu3 I see all information and all sensors.
But in my homebridge i see only temperature, humidity and brightness. No rain detection, no rainfall, no wind speed.
How can I see them too?

Any ideas?

Thanks
Peter

**Comment by MylesBorins** [comment]
hey @ppoths2297 this discussion form is for the npm registry + CLI not for homematic.
